docs/api/neurons.md

```diff
@@ -0,0 +1,80 @@
+# Kalliope API: Neurons
+
+| Method | URL              | Action                        |
+|:-------|:-----------------|:------------------------------|
+| GET    | /neurons         | Get list of installed neurons |
+| GET    | /neurons/install | Install a community neuron    |
+
+
+## List installed neurons
+
+Normal response codes: 200
+Error response codes: unauthorized(401)
+Curl command:
+```bash
+curl -i \
+--user admin:secret \
+-X GET \
+http://localhost:5000/neurons
+```
+
+Output example:
+```JSON
+{
+  "community": [
+    "gmail_checker", 
+    "wikipedia_searcher"
+  ], 
+  "core": [
+    "script", 
+    "brain", 
+    "systemdate", 
+    "neurotimer", 
+    "uri", 
+    "say", 
+    "sleep", 
+    "ansible_playbook", 
+    "neurotransmitter", 
+    "settings", 
+    "signals", 
+    "shell", 
+    "debug", 
+    "mqtt_publisher", 
+    "kalliope_version", 
+    "kill_switch"
+  ]
+}
+```
+
+## Install a neuron
+
+```bash
+curl -i -H "Content-Type: application/json" \
+--user admin:secret \
+-X POST \
+-d '
+{
+    "git_url": "https://github.com/kalliope-project/kalliope_neuron_wikipedia.git",
+    "sudo_password": "raspberry"
```

Comment:
Not very safe :).... Maybe we could extract an issue to found a better solution

Reply:
You are right.

Reply:
I opened an issue about removing the Ansible integration. It's painful to keep it in the project. And btw the last version 2.9 can even not be placed in a setup.py.
I would propose to leave this as it for this minor release and refactor everything in the next major release.